.github/workflows/verify_library_generation.yaml

```diff
@@ -45,35 +43,13 @@ jobs:
         pushd library_generation
         pip install -r requirements.in
         popd
-
-    - name: install utils (macos)
-      if: matrix.os == 'macos-12'
-      shell: bash
-      run: |
-        brew update --preinstall
-        # we need the `realpath` command to be available
-        brew install coreutils
-    - name: install docker (ubuntu)
```

Comment:
We don't need to install docker because it's included in [ubuntu-22.04](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#tools)